From dd24a69e56efb928f7a136b7018df3b6b70bab33 Mon Sep 17 00:00:00 2001
From: Sergio Pena <sergio.pena@cloudera.com>
Date: Fri, 6 Nov 2015 11:55:30 -0600
Subject: [PATCH 0386/1164] FIXUP: HIVE-10895: ObjectStore does not close
 Query objects in some calls ...

The original HIVE-10895 has 2 lines that were removed when backporting the patch
to CDH. I am adding them back because they are needed by CDH-34112: HIVE-9906

Change-Id: Ieafb3d9568257b8b22744444ae0a74a38f9d2a40
---
 .../hadoop/hive/metastore/TestObjectStore.java     |    2 ++
 1 file changed, 2 insertions(+)

diff --git a/metastore/src/test/org/apache/hadoop/hive/metastore/TestObjectStore.java b/metastore/src/test/org/apache/hadoop/hive/metastore/TestObjectStore.java
index 2c1ee76..a4f9f6c 100644
--- a/metastore/src/test/org/apache/hadoop/hive/metastore/TestObjectStore.java
+++ b/metastore/src/test/org/apache/hadoop/hive/metastore/TestObjectStore.java
@@ -76,6 +76,7 @@ public void setUp() {
     objectStore = new ObjectStore();
     objectStore.setConf(conf);
 
+    Deadline.registerIfNot(100000);
     try {
       objectStore.dropDatabase(DB1);
     } catch (Exception e) {
@@ -166,6 +167,7 @@ public void testPartitionOps() throws MetaException, InvalidObjectException, NoS
     Partition part2 = new Partition(value2, DB1, TABLE1, 222, 222, sd, partitionParams);
     objectStore.addPartition(part2);
 
+    Deadline.startTimer("getPartition");
     List<Partition> partitions = objectStore.getPartitions(DB1, TABLE1, 10);
     Assert.assertEquals(2, partitions.size());
     Assert.assertEquals(111, partitions.get(0).getCreateTime());
-- 
1.7.9.5

